app/components/task/extension-form.hbs

```diff
@@ -0,0 +1,48 @@
+<div class='extension-form__container-parent'>
+  <div role="button" class="extension-form__container-back" {{on 'click' @closeModel}}></div>
+  <div class='extension-form__container-main'>
+    <h2>{{@title}}</h2>
+    {{#if this.createExtensionRequest}}
+      <div class='extension-form__content'>
+        <h3>Create Extension Request!!!</h3>
+        <form {{on 'submit' this.submitExtensionRequest}}>
+          <label for='reason'>Reason</label>
+          <input required type='text' id='reason' name='reason' />
+          <label for='newEndsOn'>New ETA</label>
+          <p>Old ETA - {{this.oldETA}}</p>
+          <input
+            required
+            type='datetime-local'
+            id='newEndsOn'
+            name='newEndsOn'
+          />
+          <label for='title'>Title</label>
+          <input required type='text' id='title' name='title' />
+          <button type='submit'>Create</button>
```

Comment:
Can we please disable the button and add a spinner when a request is sent??
As of now, if the request takes time, the user is able to hammer the create button for that amount of time and can send the `n` number of requests. 
![image](https://user-images.githubusercontent.com/78433013/213881107-0e5376d3-a666-45f2-b312-0ddf7eac6631.png)

Reply:
Will get this done.

Reply:
Can we please disable the close modal action as well when the API call is going on ? It may lead to weird behavior that user can face.

Reply:
[Screencast from 2023-02-25 17-02-42.webm](https://user-images.githubusercontent.com/78433013/221354675-ab8c8475-4445-472d-b83f-9bd0b14bb4ce.webm)